.github/workflows/ci.yml

```diff
@@ -5,7 +5,7 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        node-version: [8, 10, 12]
+        node-version: [10, 12, 14]
```

Comment:
We just made this change in https://github.com/sveltejs/eslint-plugin-svelte3/pull/77, so if you rebase against `master` you should be able to drop this file from the PR